### PR TITLE
Functions:Introduction:Quiz

### PR DESCRIPTION
**Pull request by lindbrook**
Link in the middle of the sentence?

### CODE REVIEW

File: Functions.Rmd [hunk 1]
```diff
@@ -11,7 +11,7 @@ If you're reading this book, I expect that you've already created many R functio
 
 ### Quiz {-}
 
-Answer the following questions to see if you can safely skip this chapter. You can find the answers at the end of the chapter in [answers](#function-answers).
+Answer the following questions to see if you can safely skip this chapter. You can find the [answers](#function-answers) at the end of the chapter.
```
Comment (hadley) [inline Functions.Rmd]: We need to rewrite these to be (e.g.) in `Section \@ref(function-answer)` so they look the same in online and printed books.

**Comment by lindbrook** [comment]
Trying to avoid using "answers" twice. Not sure if this works stylistically or syntactically.

**Comment by hadley** [comment]
Maybe just `You can find the answers in \@ref(function-answer)?`. 

Alternatively, now that we have footnotes, we could put the answer to each question in its own footnote?

**Comment by lindbrook** [comment]
Probably better in a separate section (at end of chapter?). Less peaking.

**Comment by hadley** [comment]
BTW for this edition of the book (since I'm now using bookdown), I'm slowly moving all cross-links to the `Section \@(ref)` form

**Pull request merged**